doc/src/build_chain/fpga_api/api_build.md

```diff
@@ -63,14 +85,16 @@ Valid «user targets» are:
   make target                Purpose
   -------------------------- --------------------------------------------------------------------------------------
   make                       Compiles the Intel FPGA API libraries, sample applications, utilities
-  make dist                  Creates distributable tarball package intel-fpga\_0.1.0.tar.gz
-  make package               Create Redhat installer package: intel-fpga\_0.1.0\_1.x86\_64.rpm
+  make dist                  Creates distributable tarball package intel-fpga_0.1.0.tar.gz
+  make package               Create Redhat installer package: intel-fpga_0.1.0_1.x86_64.rpm
```

Comment:
Not sure about the package names, how are they updated?
Are the backslashes not required anymore?

Reply:
We dont need backslashes anymore, unless we export to LaTeX/PDF.. so i guess we should keep them.

also as you mentioned the file names and version number are outdated. i think we should configure this file with a template such that the version and package names are correct